README.md

```diff
@@ -15,7 +15,7 @@ To view our site and see your contributions, visit <https://fcc-seoul.fly.dev>
 - Husky setup (only once)
 
   ```bash
-  yarn prepare
+  yarn run postinstall
```

Comment:
I think you don't need `run` here, but not 100% sure.

Reply:
okay, neither am I. I was just following the docs.

![image](https://github.com/user-attachments/assets/8de295c4-b624-460f-96c2-aa9e2744da97)

Reply:
This seems legit to me